langchain4j-pinecone/pom.xml

```diff
@@ -24,24 +24,7 @@
         <dependency>
             <groupId>io.pinecone</groupId>
             <artifactId>pinecone-client</artifactId>
-            <version>0.6.0</version>
-            <exclusions>
-                <!-- CVE-2023-44487 -->
-                <exclusion>
-                    <groupId>io.netty</groupId>
-                    <artifactId>netty-codec-http2</artifactId>
-                </exclusion>
-                <!-- CVE-2023-3635 -->
-                <exclusion>
-                    <groupId>com.squareup.okio</groupId>
-                    <artifactId>okio-jvm</artifactId>
-                </exclusion>
-                <!-- CVE-2020-29582 -->
-                <exclusion>
-                    <groupId>org.jetbrains.kotlin</groupId>
-                    <artifactId>kotlin-stdlib</artifactId>
-                </exclusion>
-            </exclusions>
+            <version>1.0.0</version>
```

Comment:
Thank you for migrating to 1.0.0! 🤗

Reply:
Pinecone API 1.1.0 is out,  it looks like its only added a new feature (https://github.com/pinecone-io/pinecone-java-client/compare/v1.0.0...v1.1.0) and a couple of bug fixes, might be worth upgrading while we have eyes on it.